testsuite/libra-fuzzer/README.md

```diff
@@ -86,7 +86,7 @@ See [Google OSS-Fuzz's documentation on reproducing bugs](https://google.github.
 To test coverage of our fuzzers you can run the following command with [grcov](https://github.com/mozilla/grcov):
 
 ```
-CORPUS_PATH=fuzz/corpus cargo xtest --html-cov-dir <some path for html output> -p libra-fuzzer -- coverage
+RUSTFLAGS='--cfg feature="fuzzing"' CORPUS_PATH=fuzz/corpus cargo xtest --html-cov-dir <some path for html output> -p libra-fuzzer -- coverage
```

Comment:
we should code mod this to testing since it is used for both purposes.

Reply:
Pretty sure that xtest, which turns on all features makes this unnecessary.   Also confident that x will ignore this set of flags and not pass them through.   Confident that this is a no op.